_posts/0000-01-03-cschwab9.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Something funny here
 
 ![kimonotocat](https://octodex.github.com/images/kimonotocat.png)
```

Comment:
Maybe use this image instead?

![turkey](https://octodex.github.com/images/thanktocat.png)